pyroute2/ipset.py

```diff
@@ -161,12 +163,24 @@ def create(self, name, stype='hash:ip', family=socket.AF_INET,
         if forceadd:
             cadt_flags |= IPSET_FLAG_WITH_FORCEADD
 
+        if 'bitmap' in stype and bitmap_range is None:
+            raise ValueError('Missing value bitmap_range')
+
         data = {"attrs": [["IPSET_ATTR_CADT_FLAGS", cadt_flags],
                           ["IPSET_ATTR_MAXELEM", maxelem]]}
         if hashsize is not None:
             data['attrs'] += [["IPSET_ATTR_HASHSIZE", hashsize]]
         if timeout is not None:
             data['attrs'] += [["IPSET_ATTR_TIMEOUT", timeout]]
+        if bitmap_range is not None and 'bitmap' in stype:
+            # Set the bitmap range A bitmap type of set
+            # can store up to 65536 entries
+            try:
+                f, t = [int(item) for item in bitmap_range.split('-')]
+            except:
+                raise TypeError('Invalid bitmap_range')
+            data['attrs'] += [['IPSET_ATTR_PORT_FROM', f]]
```

Comment:
Your patch will work only on bitmap:port ipset, since we will need IPSET_ATTR_IP_FROM and IPSET_ATTR_IP_TO for bitmap:ip and bitmap:ip,mac. You should update your commit message, and probably rename the bitmap_range variable to something like bitmap_ports_range